### DISCUSSION TITLE
Welcome to enhancements Discussions!

### DISCUSSION THREAD

**Discussion by jwmatthews** [konveyor/enhancements#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shawn-hurley** [comment]
Hello Everyone, 

My name is Shawn Hurley and I work on crane and oadp at Red Hat.

**Comment by jmontleon** [comment]
Hi, my name is Jason Montleon and I work at Red Hat on Crane / MTC. I also sometimes help out with OADP.

**Comment by sseago** [comment]
Hi,

My name is Scott Seago and I work on crane, velero, and OADP at Red Hat.

**Comment by mrizzi** [comment]
Hi all :wave:
I'm Marco Rizzi and I'm a Red Hat engineer working on Tackle and [Windup](https://github.com/windup).

**Comment by ascheman** [comment]
Hi,

my name is Gerd Aschemann. I am a freelance consultant currently on a gig with a consulting company to help a large Germany-based international company towards their cloud migrations. We are happy we found Konveyor, as it supports our approach to industrialize as much as possible (we have a few hundred projects as migration candidates).
Therefore, we have started to work with the tools provided by Konveyor, ran into some problems and were in particular happy to find such a welcoming and helpful community here.
Kudos already to @jwmatthews, @jmle, @pranavgaikwad, and @shawn-hurley for the great discussions on Slack and in the tickets we already created.
